docs/reference-guides/interactivity-api/core-concepts/how-the-interactivity-api-works

```diff
@@ -0,0 +1,193 @@
+
+## Server-side processing begins
+
+### 1. Initial processing
+
+WordPress handles the initial request to the `/interactive` page, triggering all the usual hooks during processing.
+
+### 2. HTML DOM processing
+
+At this stage, the current theme generates the markup for the request. During this process, the section containing the interactive accordion block is rendered:
+
+```php
+// render.php
+<?php wp_interactivity_state('accordion', [
+  'isOpen'             => false,
+  'isLastItemSet'      => false,
+  'lastOpenedItemName' => "",
+]); ?>
+
+<div class="accordion"
+     data-wp-interactive="accordion"
+<?php echo wp_interactivity_data_wp_context(["openItemMessage" => __('Current open item is', 'domain'), "closedItemMessage" => __('Items are closed.', 'domain'), "lastOpenItemMessage"=> __('Last opened item is', 'domain'),]); ?>
+      data-wp-class--dark="state.isDark">
+    <div class='accordion__panel'>
+        <div class='accordion__heading-opened' data-wp-bind--hidden="!state.isOpen">
+            <span data-wp-text="context.openItemMessage"></span>&nbsp;
+            <span class='accordion__open-item-name' data-wp-text="state.lastOpenedItemName"></span>
+        </div>
+        <div class='accordion__heading-closed' data-wp-bind--hidden="state.isOpen">
+            <span data-wp-text="context.closedItemMessage"></span>
+            <p class='accordion__closed-item' data-wp-bind--hidden="!state.isLastItemSet">
+                <span data-wp-text="context.lastOpenItemMessage"></span>&nbsp;
+                <span class='accordion__closed-item-name' data-wp-text="state.lastOpenedItemName"></span>
+            </p>
+        </div>
+    </div>
+
+    <div class='accordion__item' data-wp-context='{"isItemClosed":true,"itemName":"First"}'>
+        <p class='accordion__item-title' data-wp-on--click="actions.toggleItem" data-wp-text="context.itemName"></p>
+        <div class='accordion__item-content' data-wp-bind--hidden="context.isItemClosed">Content of the first item</div>
+    </div>
+
+    <div class='accordion__item' data-wp-context='{"isItemClosed":true, "itemName":"Second"}'>
+        <p class='accordion__item-title' data-wp-on--click="actions.toggleItem" data-wp-text="context.itemName"></p>
+        <div class='accordion__item-content' data-wp-bind--hidden="context.isItemClosed">Content of the second item
+        </div>
+    </div>
+</div> 
+```
+
+Note: The `wp_interactivity_state` and `wp_interactivity_data_wp_context` calls are necessary for this example but are not required for the Interactivity API to function in general. 
+On the other hand, the `data-wp-interactive="accordion"` directive is required and informs the Interactivity API that the element and its nested DOM elements should be recognized by the API.
+
+The following will happen:
+
+#### 2.1) The `wp_interactivity_state` call initializes some server-side state for the block.
```

Comment:
We are now referring to the state as global state to differentiate it from context, which is now called local context. More information [here](https://developer.wordpress.org/block-editor/reference-guides/interactivity-api/core-concepts/undestanding-global-state-local-context-and-derived-state/).